node/src/chain_spec.rs

```diff
@@ -87,7 +77,7 @@ where
 }
 
 pub fn development_config(id: ParaId) -> ChainSpec {
-    let mut properties = Map::new();
+    let mut properties = sc_chain_spec::Properties::new();
     properties.insert("tokenSymbol".into(), "DHX".into());
```

Comment:
just a thought, do we want to change this from `DHX` to `UNIT`?

Reply:
Yes @ltfschoen  We can change this if we are using it only for local testing. We are not using this for production right?

Reply:
please check with Ilya and Christian first, i'm not sure what they are already using for production

Reply:
@festelo @cgroeschel Could you please provide your thoughts?

Reply:
The question is which symbol to use in the DataHighway Parachain? I think previously we just used DHX everywhere and then changed to chain specific ones at some point in order to match the chain, eg. HBR in Harbour, DEV in brickable (I believe) and DHX in westlake standalone chains. Not so sure what we do about the token symbols for the several parachains/relaychains, eg. if we can just use DHX everywhere?